spec/models/public_body_spec.rb

```diff
@@ -532,7 +532,6 @@ def set_default_attributes(public_body)
   before(:each) do
     # Make sure that there's no internal_admin_body before each of
     # these tests:
-    PublicBody.connection.delete("DELETE FROM public_bodies WHERE url_name = 'internal_admin_body'")
```

Comment:
Just noting that this probably needs a replacement – the line below deletes the translation, but the `PublicBody` record will still exist.